packages/formvuelate/src/features/ParsedSchema.js

```diff
@@ -89,11 +90,27 @@ export default function useParsedSchema (refSchema, model) {
       }
     }
 
+    const injectedLookupSchemaForm = inject(LOOKUP_PARSE_SUB_SCHEMA_FORMS, null)
```

Comment:
Just wanted to note this logs a "inject can be only used inside setup" warning in unit tests for `ParsedSchema`. I think this is fine and can be safely ignored.